xbmc/settings/AdvancedSettings.cpp

```diff
@@ -1199,6 +1199,12 @@ void CAdvancedSettings::ParseSettingsFile(const std::string &file)
       m_seekSteps.push_back(atoi((*it).c_str()));
   }
 
+  pElement = pRootElement->FirstChildElement("suspend");
+  if (pElement)
+  {
+    XMLUtils::GetBoolean(pElement, "waitnetonwakeup", m_waitForNetAfterWakeUp);
+  }
```

Comment:
I thought you were a bot at first since the comments were so methodical and clean. +1